src/screens/UserPortal/Donate/Donate.tsx

```diff
@@ -128,14 +128,36 @@ export default function donate(): JSX.Element {
   }, [donationData]);
 
   const donateToOrg = (): void => {
+    // check if the amount is non empty and is a number
+    if (amount === '' || Number.isNaN(Number(amount))) {
+      toast.error(t(`invalidAmount`));
+      return;
+    }
+
+    // check if the amount is non negative and within the range
+    const minDonation = 1;
+    const maxDonation = 10000;
```

Comment:
1. Why was this value selected as the maximum? 
2. Does it take into consideration all exchange rates across the world? 
    - For example 10,000,000 Venezuelan Bolivares are worth just under US$3.00 at current exchange rates.

Reply:
well, for that I would need @varshith257's help because he suggested that.

Reply:
Please use your initiative and create a usable solution.

Reply:
I think it would be better if you guys create a new issue for that. The reason is we need proper discussion for the max limit according to our app over all things.